Directory.Build.props

```diff
@@ -1,7 +1,6 @@
 <Project>
     <PropertyGroup>
-        <TargetFrameworks>net5.0;net6.0;net7.0;</TargetFrameworks>
-        <Platform>x64</Platform>
+        <TargetFrameworks>net6.0;net7.0;net8.0</TargetFrameworks>
```

Comment:
no need ot add net8.0 or net7.0 unless you can do something speciific with a TFM-specific build see https://github.com/serilog/serilog/issues/1970. (i.e. I'd just leave `net6.0` unless there are new things that 7 or g can let you proide specific behavior via a custom pakage

Reply:
Thanks for the input and yes net 8 is fundamental in supporting a lot of new features going forward.